Mail/smtp.php

```diff
@@ -305,7 +305,7 @@ protected function send_or_fail($recipients, $headers, $body)
                 $params .= ' ' . $key . (is_null($val) ? '' : '=' . $val);
             }
         }
-        if (PEAR::isError($res = $this->_smtp->mailFrom($from, ltrim($params)))) {
+        if (PEAR::isError($res = $this->_smtp->mailFrom($from, ltrim($params ?? '')))) {
```

Comment:
I think we'd like to keep PHP5 compatibility here. Just change the `$params` variable initialization on line 302 instead.

Reply:
That is nicer indeed. I changed the initialization of the $params variable as you suggested.